server.go

```diff
@@ -8,20 +8,26 @@ import (
 	"net/http"
 	"bytes"
 	"strings"
-
-	"github.com/clarifai/clarifai-go"
 	"github.com/gin-gonic/gin"
+	"os"
+	"net/url"
 )
 
 const (
 	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
```

Comment:
I see the reason now. The current API is in v2, see https://developer.clarifai.com/guide/ . And the old v1 is referenced here https://developer.clarifai.com/guide-v1/ . I think we should stick with v2